airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -174,18 +209,27 @@ def submit_job(self, context: Context):
             self.job_definition,
             self.job_queue,
         )
-        self.log.info("AWS Batch job - container overrides: %s", self.overrides)
+
+        if self.container_overrides:
+            self.log.info("AWS Batch job - container overrides: %s", self.container_overrides)
+        if self.array_properties:
+            self.log.info("AWS Batch job - array properties: %s", self.array_properties)
+        if self.node_overrides:
+            self.log.info("AWS Batch job - node properties: %s", self.node_overrides)
```

Comment:
should this always be printed? Are there any security or verbosity concerns here?

Reply:
This settings not intend to use for pass sensitive information, same info stored into the rendered templates and in AWS Console Batch Job Details.